benchmarks/span/utils.py

```diff
@@ -0,0 +1,26 @@
+import random
```

Comment:
why the file name change?

Reply:
consistency with how we often refer to a "utility file" but not much more than that.